Deceive/MainController.cs

```diff
@@ -87,6 +88,15 @@ private void UpdateUI()
             {
                 Checked = _overlay != null
             };
+            
+            var mucMenuItem = new MenuItem("Lobby Chat enabled", (a, e) =>
```

Comment:
Will rename this to "Enable lobby chat" after merging since I think it makes more sense.